client/src/components/molecules/PrivateRoute.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Route, Redirect } from 'react-router-dom';
+
+const PrivateRoute = ({ component: Component, ...rest }) => {
+  const isAuthorized = () => true;
```

Comment:
why not just a consttant ?

Reply:
Will be adding an Authorization logic Later, hence used it here as a function, so that everywhere down the line it is treated as a function and not as a value.